pkgs/applications/window-managers/wayfire/plugins.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, meson, ninja
+, fetchFromGitHub, wayfire, pkgconfig
+, wlroots, wayland, wayland-protocols
+, cairo, glibmm, iio-sensor-proxy
+, libxkbcommon, glm, librsvg
+, boost, gtkmm3, udev
+, libinput
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wayfire-plugins-extra";
+  version = "b2036fc4e5935bf459b1779b89b59a73be92da9f";
+
+  src = fetchFromGitHub {
+    owner = "WayfireWM";
+    repo = "wayfire-plugins-extra";
+    rev = version;
+    sha256 = "sha256-u1gOd6Sp2hDgOA+gln6C+ADyzrI0HrGQFu5L0zFpZuE=";
+    fetchSubmodules = true;
```

Comment:
Please do not use the included dbus.

Reply:
This a plugin for dbus interface it is not dbus. I create a seperate package for it.